opensrp-reveal/src/test/java/org/smartregister/reveal/application/RevealApplicationTest.java

```diff
@@ -15,10 +22,12 @@
 
     private RevealApplication revealApplication;
     private UserService userService;
+    Context context;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/OpenSRP/opensrp-client-reveal/pullRequest?prid=4217760)